susemanager/bin/pg-migrate-12-to-13.sh

```diff
@@ -124,6 +124,8 @@ chown postgres:postgres /var/lib/pgsql/data/*
 
 echo "`date +"%H:%M:%S"`   Starting spacewalk services..."
 systemctl start postgresql
+echo "Reindexing database. This may take a while, please do not cancel it!"
+database=$(sed -n "s/^\s*db_name\s*=\s*\([^ ]*\)\s*$/\1/p" /etc/rhn/rhn.conf)
+spacewalk-sql --select-mode - <<<"REINDEX DATABASE ${database};"
 spacewalk-service start
```

Comment:
I would check if the query was successful. We don't want to start suma if reindexing failed.

Reply:
Makes sense, but please check if `spacewalk-sql` is returning error codes according to the result of the query!

Reply:
good idea.